core/storage/classifier/gae_models_test.py

```diff
@@ -111,6 +111,115 @@ def test_query_new_and_pending_training_jobs(self):
             feconf.TRAINING_JOB_STATUS_PENDING)
         self.assertEqual(offset, 2)
 
+    def test_query_new_and_pending_training_jobs_with_multiple_fetches(self):
```

Comment:
Test names should explain what's expected to happen (& in some ways that's more important than the state that leads to that outcome). Please update this to include those details since it actually doesn't seem clear to me from the current wording.

If it helps, see also: https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests.

Reply:
Done. PTAL